README.md

```diff
@@ -2,6 +2,7 @@
 
 ## Libraries
 
+- [node.js and npm](https://nodejs.org/en/) - Package manager and server workhorse (minimum **version 17**)
```

Comment:
Minimum version is the LTS version (16.1X.X)

Reply:
Done